.classpath

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please, remove from git files which not related to the module (eclips config files, .project etc.)